core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/aggregate/SpecializedAggregateOpDesc.scala

```diff
@@ -71,6 +72,7 @@ class SpecializedAggregateOpDesc extends AggregateOpDesc {
     }
     Schema
       .newBuilder()
+      .add(ProgressiveUtils.insertRetractFlagAttr)
```

Comment:
Alternatively, we can have a `Builder.allowRetract()` to add this attribute internally for users?